tests/e2e/cucumber/features/smoke/upload.feature

```diff
@@ -44,10 +44,12 @@ Feature: Upload
       | PARENT     | folder |
       # Coverage for bug: https://github.com/owncloud/ocis/issues/8361
       | comma,.txt | file   |
-  #  currently upload folder feature is not available in playwright
-  #  And "Alice" uploads the following resources
-  #    | resource |
-  #    | PARENT   |
+
+    # https://github.com/owncloud/web/issues/6348
+    # currently upload folder feature is not available in playwright
+    # And "Alice" uploads the following resources
+    #  | resource |
+    #  | PARENT   |
```

Comment:
will this work? because folder `PARENT` is already there. Also, do `PARENT` has any empty folder in it?